src/tests/topp/THIRDPARTY/third_party_tests.cmake

```diff
@@ -102,6 +102,10 @@ OPENMS_FINDBINARY(THERMORAWFILEPARSER_BINARY "ThermoRawFileParser.exe" "ThermoRa
 # LuciPhor2
 OPENMS_FINDBINARY(LUCIPHOR_BINARY "luciphor2.jar" "LuciPHOr2")
 
+#------------------------------------------------------------------------------
+# CometAdapter (Used in DatabaseSuitability)
+OPENMS_FINDBINARY(COMET_ADAPTER_BINARY "CometAdapter" "CometAdapter")
```

Comment:
I think I am fine with this fix but would like to hear what @jpfeuffer thinks? Basically, checking if any OpenMS executable is found would work as well here?